docs/rfcs/component-universal-telemetry.md

```diff
@@ -54,6 +54,7 @@ Spans, data points, and log records should carry the following attributes:
 - `otelcol.component.id`: The component ID
 - `otelcol.signal`: `logs`, `metrics` `traces`
 - `otelcol.signal.output`: `logs`, `metrics`, `traces`, `profiles`
+- `otelcol.pipeline.id.output`: The pipeline ID to which the data is sent
```

Comment:
I've proposed this as a data point attribute in https://github.com/open-telemetry/opentelemetry-collector/pull/12815 and am working on corresponding changes.

Reply:
```suggestion
```